### PR TITLE
Update dom-bind template for Pomyler 2 (For #25)

### PR DESCRIPTION
**Pull request by PatrykSzwer**
I'm not sure if I should target this PR to `develop` branch or `master`. I've made it to the default `develop` right now. Let me know If I should change that.

### CODE REVIEW

File: src/Starcounter.VS.HTMLTemplateItemTemplate/HTMLTemplate.html [hunk 1]
```diff
@@ -4,7 +4,10 @@
     see: https://www.polymer-project.org/1.0/docs/api/elements/dom-bind
```
Comment (alshakero) [inline src/Starcounter.VS.HTMLTemplateItemTemplate/HTMLTemplate.html]: Could you update the comment to `<dom-bind>` and the link to Polymer 2.0's dom-bind?

File: src/Starcounter.VS.HTMLTemplateItemTemplate/HTMLTemplate.html [hunk 2]
```diff
@@ -4,7 +4,10 @@
...
 -->
 <template>
-    <template is="dom-bind">
-        
-    </template>
+    <dom-bind>
+        <!-- `is` attribute is optional for Polymer 1 backwards compatibility -->
```
Comment (alshakero) [inline src/Starcounter.VS.HTMLTemplateItemTemplate/HTMLTemplate.html]: It's my fault, but maybe Polymer 1.x is better than Polymer 1.

**Review by alshakero — CHANGES_REQUESTED** [review]

**Comment by warpech** [comment]
In response to @alshakero's review, I'd rather have just one comment in this template:

```
<!--
    <dom-bind><template is="dom-bind"> is a Polymer 2 auto-binding template
    with a compatibility fallback for Polymer 1. See:
    https://www.polymer-project.org/2.0/docs/devguide/templates
-->
```

**Comment by PatrykSzwer** [comment]
I've made a change due to @warpech comment. Please take a look.

**Review by alshakero — APPROVED** [review]

**Comment by PatrykSzwer** [comment]
Due to @warpech comment: https://github.com/Starcounter/Starcounter.VisualStudio/issues/25#issuecomment-356010111
@ubbeK I'm pinging You to release a new version of the extension. 😃

**Comment by warpech** [comment]
Release in Visual Studio marketplace is tracked in https://github.com/Starcounter/Starcounter.VisualStudio/issues/27

**Pull request merged**